src/lib/renderer/__tests__/RatingDecorator.test.ts

```diff
@@ -58,6 +58,16 @@ describe("RatingDecorator test", () => {
         expectNotToBeRatingElement(rendered.pop());
     });
 
+    it("do not render rating if elements is empty", () => {
+        const renderer = new RatingDecorator(new ContentCentricRenderer(), RatingRenderStrategy.ONLY_ENABLED_RATINGS);
+        const specification = RatingTestSpecGenerator.generate(false, false);
```

Comment:
Named arguments sind wahrscheinlich kein thing out of the box in TS, oder? Wäre cool wenn das gehen würde, sonst ist schwierig zu erkennen was `false, false` bedeutet.

https://medium.com/better-programming/named-parameters-in-typescript-e32c763d2b2e

Reply:
Ist ein guter Punkt - mag selbst die random boolean Flags nicht, aber hab mich im Kontext von einem Test halt dazu hinreißen lassen. ;-)